### PR TITLE
Update Im Notes for #369 Redirects, #377 Path mappings

### PR DESCRIPTION
**Pull request by neilsjefferies**
#369 OCFL Does not support redirects
#377 Add para to 2.2 on PID mapping algorithms

### CODE REVIEW

File: draft/implementation-notes/index.html [hunk 1]
```diff
@@ -398,7 +398,15 @@ <h2>Storage Root Hierarchy</h2>
                 └── ...
             </pre>
           </li>
-      </ul>
+        </ul>
+      <p>
+	Some identifier schemes may require transformation before such approaches can be used effectively. A simple
+	example would be sequentially assigned identifiers, which would not distribute objects within the filesystem
+	evenly, or variable length identifers. Hash functions may be used to provide a unidirectional mapping between
```
Comment (rosy1280) [inline draft/implementation-notes/index.html]: please drop the `variable length identifiers` part.

File: draft/implementation-notes/index.html [hunk 2]
```diff
@@ -398,7 +398,15 @@ <h2>Storage Root Hierarchy</h2>
...
+	URI/PID and filesystem path as required by the OCFL. Encryption algorithms may be used to provide a 
+	bi-directional mapping which may be a useful aid to recoverability. Relevant details should be recorded in
```
Comment (rosy1280) [inline draft/implementation-notes/index.html]: s/recoverability/human readability

File: draft/implementation-notes/index.html [hunk 3]
```diff
@@ -523,12 +531,11 @@ <h2>File Purging</h2>
...
       </p>
       <p>
-        The new object need not have the same identifier as the original object. In this case, the deleted object can be
-        replaced by a "stub" object with the original identifier and location in the OCFL Storage Root. This object is
-        a standard OCFL object that just contains brief information that redirects users and software to the new
-        version - possibly with an indication of why the new object was created, if appropriate. The OCFL does not
-        define this stub object in any way - its structure, interpretation and handling are entirely client dependent,
-        but ideally some elements should be human readable for rebuildability.
+        The new object need not have the same identifier as the original object. In this case, the deleted object may be
+        replaced by a new object with the original identifier and location in the OCFL Storage Root. This object is
```
Comment (rosy1280) [inline draft/implementation-notes/index.html]: may be

File: draft/implementation-notes/index.html (hunk 2 above)
Comment (awoods) [inline draft/implementation-notes/index.html]: `recorded` -> `referenced`

File: draft/implementation-notes/index.html [hunk 4]
```diff
@@ -523,12 +531,11 @@ <h2>File Purging</h2>
...
+        a standard OCFL object with content that redirects users and software to the new version - possibly with an 
+	indication of why the new object was created, if appropriate. The OCFL does not define redirect mechanisms,
```
Comment (awoods) [inline draft/implementation-notes/index.html]: indentation

File: draft/implementation-notes/index.html (hunk 3 above)
Comment (rosy1280) [inline draft/implementation-notes/index.html]: s/with the/using the

Comment (awoods) [inline draft/implementation-notes/index.html]: `new object` -> `"stub" object`

**Pull request merged**